src/main/java/Akinator.java

```diff
@@ -0,0 +1,210 @@
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+import java.util.concurrent.TimeUnit;
+
+
+public class Akinator {
+    public static WebDriver driver;
+
+    public static void main(String[] args) throws InterruptedException, NoSuchElementException {
+        initiateGame();
+        answerQuestions();
+        confirmYourHero();
+
+        endGame();
+    }
+
+    //    Різниця між implicitlyWait & sleep. Розібратися що коли краще використовувати для економії часу на проходження
+    public static void initiateGame() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
+
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        secWait(10);
+        driver.get("https://ru.akinator.com");
+        clickCss("a[href='/game']");
+    }
+
+    private static void secWait(int sec) {
+        driver.manage().timeouts().implicitlyWait(sec, TimeUnit.SECONDS);
+    }
+
+    public static void endGame() {
+        driver.quit();
+    }
+
+    public static void showFiveOptions() {
+
+        String question = driver.findElement(By.xpath("//p[@class='question-text']")).getText();
+        System.out.println("Question: " + question);
+        System.out.println("Варианты ответа:");
+        String first = textCss("a[id='a_yes']");
+        System.out.println(first + " - 1");
+
+        System.out.println(textCss("a[id='a_no']") + " - 2");
+        System.out.println(textCss("a[id='a_dont_know']") + " - 3");
+        System.out.println(textCss("a[id='a_probably']") + " - 4");
+        System.out.println(textCss("a[id='a_probaly_not']") + " - 5");
+        try {
+            System.out.println(textCss("span[class='back-button-text']") + " - 6");
+        } catch (Exception e){
+            System.out.print("");
+        }
+    }
+
+    public static int showTwoOptions() {
+        String yes = textCss("a[id='a_propose_yes']");
+        String no = textCss("a[id='a_propose_no']");
+        System.out.println("--------------------");
+        System.out.println("| "+yes+" - 1 | "+no+" - 2 |");
+        System.out.println("--------------------");
+
+
+        Scanner scanner = new Scanner(System.in);
+        int option = scanner.nextInt();
+
+        switch (option) {
+            case 1:
+                return 1;
+            case 2:
+                return 2;
+            default:
+                System.out.println("Этого варианта не существует. Попробуйте еще раз. Спасибо!");
+                showTwoOptions();
+        }
+        return 0;
+    }
+
+    public static void answerQuestions() throws InterruptedException, NoSuchElementException {
+        TimeUnit.SECONDS.sleep(5);
```

Comment:
Could you please extract this logic to separate method and catch exceptions there? Don't forget to remove **throws ...**)